Signum.React/Filters/SignumExceptionFilterAttribute.cs

```diff
@@ -30,11 +30,13 @@ public class SignumExceptionFilterAttribute : IAsyncResourceFilter
 
         public static readonly List<Type> AvoidLogException = new List<Type> { typeof(OperationCanceledException) };
 
+        public static Func<Exception, HttpError> CustomHttpErrorFactory = ex => new HttpError(ex);

```

Comment:
How you used this extension point?

Reply:
Hi Olmo,

we use this extension point to explicitly control the information that will be included in an `HttpError`. 

The `SignumExceptionFilterAttribute` will return the following information by default:

- ExceptionMessage
- ExceptionType
- ExceptionId
- StackTrace
- InnerException

This extension point might be very useful for every application, that needs a fine grained control over the returned information to be f.e. compliant to strict security standards.